config/webpack.config.js

```diff
@@ -27,21 +27,14 @@ plugins = [
   new CopyWebpackPlugin([
     {
       from: 'src/images',
-      to: 'images/',
-      flatten: true
```

Comment:
The default is `flatten: false`: https://webpack.js.org/plugins/copy-webpack-plugin/#patterns